internal/proto/scan.go

```diff
@@ -105,3 +106,63 @@ func Scan(b []byte, v interface{}) error {
 			"redis: can't unmarshal %T (consider implementing BinaryUnmarshaler)", v)
 	}
 }
+
+
+// Scan a string slice into a custom container
+// Example:
+// var container []YourStruct; ScanSlice([]string{""},&container)
+// var container []*YourStruct; ScanSlice([]string{""},&container)
+func ScanSlice(sSlice []string, container interface{}) error {
+	val := reflect.ValueOf(container)
```

Comment:
I'll check immediately, but I'm not sure what is "errTyp" flag, can you show me an example? My native language is not English and some description may has another meaning for me.
Thx.